graphql/schema.test.js

```diff
@@ -0,0 +1,6 @@
+describe('schema generation', () => {
+  test('should create schema', () => {
+    const schema = require('./schema')
```

Comment:
Schema is used for server side generation. To generate data at build time you need active graphql endpoint but next js doesn't start api routes until it's finished building the app.

Reply:
I think we should disable test for schemas. It doesn't make sense to test this.

Reply:
@Ulisseus on `package.json` you can add the file on the ignore coverage path list to disable required tests